lib/Auth/Process/RestoreAcrs.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Module\perun\Auth\Process;
+
+use SimpleSAML\Auth\ProcessingFilter;
+use SimpleSAML\Logger;
+use SimpleSAML\Module\perun\Disco;
+
+/**
+ * Auth proc filter, which should be used if Proxy provides MFA, and wants to first pass the request to the upstream
+ * IdP. The Perun Disco page, tries to add new ACRs to the request, if MFA has been requested, to prevent the IdP from
+ * failing. As a result, we then get modified requested ACRs, which should be restored to the previous (original) state
+ * using this authproc filter. It should be run on one of the first places of the IdP authproc chain.
+ */
+class RestoreAcrs extends ProcessingFilter
```

Comment:
```suggestion
class multifactorAcrs extends ProcessingFilter
```
It's just that `RestoreAcrs::storeAcrs` feels weird.